lib/vjs-hls.js

```diff
@@ -29,8 +30,50 @@ var attachVideojsStreamrootProvider = function (window, videojs, Hls) {
             _hls.nextLevel = qualityId;
         }
 
-        function _onError(event, data) {
-            console.error('HLS.js error: ' + data.type + ' - fatal: ' + data.fatal + ' - ' + data.details);
+        function _onError(event, data, tech, errorCounts) {
+            var error = {
+              message: ('HLS.js error: ' + data.type + ' - fatal: ' + data.fatal + ' - ' + data.details),
+            };
+            console.error(error.message);
+
+            // increment/set error count
+            errorCounts[data.type] ? errorCounts[data.type] += 1 : errorCounts[data.type] = 1;
+            
+            // implement simple error handling based on hls.js documentation (https://github.com/dailymotion/hls.js/blob/master/API.md#fifth-step-error-handling)
+            if (data.fatal) {
+                switch (data.type) {
+                    case Hls.ErrorTypes.NETWORK_ERROR:
+                        console.info("bubbling network error up to VIDEOJS");
+                        error.code = 2;
+                        tech.error = function() { return error; };
+                        tech.trigger('error');
```

Comment:
FYI:  I was first trying to use `_hls.startLoad();` the first time a network error happens (as suggested here https://github.com/dailymotion/hls.js/blob/master/API.md#error-recovery-sample-code), but when I did that the network error was not triggered the second time, so there was no way to bubble it up to VIDEOJS and handle it there.